src/Bundler.js

```diff
@@ -332,14 +335,28 @@ class Bundler extends EventEmitter {
       let thrown = err;
 
       if (thrown.message.indexOf(`Cannot find module '${dep.name}'`) === 0) {
+        let isLocalFile = dep.name.startsWith('.');
+        // Attempt to install missing npm dependencies
+        if (!isLocalFile && this.options.autoinstall) {
+          logger.status(emoji.progress, `Installing ${dep.name}...`);
+          await installPackage(
```

Comment:
Might be time for a config object to `installPackage`... 5 parameters!

Reply:
Done